src/ophyd_async/panda/writers/_hdf_writer.py

```diff
@@ -203,9 +203,11 @@ async def collect_stream_docs(
         # TODO: fail if we get dropped frames
         if indices_written:
             if not self._file:
+                dir = Path(await self.panda_device.data.hdf_directory.get_value())
+                file = Path(await self.panda_device.data.hdf_file_name.get_value())
```

Comment:
There is already a PV that gives this:
https://github.com/PandABlocks/PandABlocks-ioc/blob/8e25d64cc97cfbf242a84c6ccbe600a39ed41c41/src/pandablocks_ioc/_hdf_ioc.py#L411-L413

Reply:
so
```python
    hdf_full_file_path: SignalR[str]
```
could be added alongside:
https://github.com/bluesky/ophyd-async/blob/b59d1e21c91a2697c36e519393f69a29b7601978/src/ophyd_async/panda/_common_blocks.py#L11-L12